showcase/app/templates/components/form/super-select.hbs

```diff
@@ -0,0 +1,1037 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form::SuperSelect Components"}}
+
+<BasicDropdownWormhole />
+
+<Shw::Text::H1>Form::SuperSelect</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Single selection</Shw::Text::H2>
+
+  <Shw::Text::H3>Interaction status</Shw::Text::H3>
+
+  <Shw::Grid @columns={{3}} as |SG|>
```

Comment:
I'm not sure if this is something we need to fix. The width of the SuperSelect is the full width of its container by default and setting a placeholder, pre-selecting an option, or using it as a field with a visible label all increase the minimum width so it seems this would only be an issue if the container collapses down or doesn't have a width and doesn't have a placeholder or other content.

We could set a minimum width but I'm not sure what a good value would be.

Reply:
By default, the native form select is as wide as the widest option it contains even if it forces the select to overflow its container. (The select width can be constrained by adding custom CSS.)

Reply:
I am resolving this for now